xilem/src/widget_view.rs

```diff
@@ -46,6 +46,36 @@ pub trait WidgetView<State, Action = ()>:
     {
         transformed(self).transform(by)
     }
+
+    /// Set a [Property] on this view, when the underlying widget [supports](HasProperty) it.
```

Comment:
Either Property should be `Property` or shouldn't be capitalised, I think. See #1187

Reply:
I used `Property`.